.github/workflows/cd-syft.yml

```diff
@@ -131,6 +131,35 @@ jobs:
             echo "github_release_version=$(python packages/grid/VERSION)" >> $GITHUB_OUTPUT
           fi
 
+      # Checkout Infra repo (nested)
+      - name: Checkout Infra Repo
+        uses: actions/checkout@v3
+        with:
+          repository: OpenMined/infrastructure
+          token: ${{ secrets.InfraRepoToken }}
+          path: infrastructure
+
+      # This step will copy the generated K8s manifest files to the correct directory in Infra repo
+      - name: Copy files to Repo B
+        run: |
+          if $is_pre_release; then
+            rm -rf infrastructure/gitops/environments/pre-release/.
+            cp -R packages/grid/helm/syft/. packages/grid/helm/manifest.yaml infrastructure/gitops/environments/pre_release/
+          else
+            rm -rf infrastructure/gitops/environments/stable/.
+            cp -R packages/grid/helm/syft/. packages/grid/helm/manifest.yaml infrastructure/gitops/environments/stable/
```

Comment:
Typo: should be `packages/grid/helm/manifests.yaml`

Reply:
Done 👍